manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -133,7 +135,8 @@ podCountLimit: "100"
 secretAnnotations: {}
 enableRuntimeDefaultSeccompProfile: true
 controllerManager:
-  exemptNamespaces: []
+  exemptNamespaces:
```

Comment:
For this, I was thinking adding a conditional here https://github.com/open-policy-agent/gatekeeper/blob/master/manifest_staging/charts/gatekeeper/templates/gatekeeper-validating-webhook-configuration-validatingwebhookconfiguration.yaml#L32 to reduce the impact as this could have backward compatibility issues if users do not currently exempt kube-system. But I'm ok if we want add it directly to values.yaml with the proper notice in the release notes. WDYT @sozercan @maxsmythe?

Reply:
@sathieu do you have any preference here?

Reply:
@ritazh I prefer adding it to `values.yaml`, which is more explicit (with release notes of course). Using logic in templates makes it hard to understand for the users.

NB: I have no urgent need in this PR, as this is only about changing default values, which [I override](https://gitlab.com/kubitus-project/kubitus-installer/-/blob/d6d163b02d9a6b02d9bd539d34e307f6d9ee4caf/roles/gatekeeper/templates/values.yaml.j2#L18-19).

Reply:
Sorry for the delay. I'm kind of borderline ok with merging this with proper release notes. WDYT? @sozercan @maxsmythe 
The impact is kube-system will get the `admission.gatekeeper.sh/ignore=extra-namespaces` label by default.
https://github.com/open-policy-agent/gatekeeper/blob/master/charts/gatekeeper/templates/namespace-post-install.yaml#L70-L76

Reply:
No problem @ritazh.

I'm also not sure that this is a good idea. It was not enough for us, as some cluster-scoped resources (namely `nodes` and `ciliumnodes`) were preventing the cluster to recover (See https://gitlab.com/kubitus-project/kubitus-installer/-/commit/5320a21fdb2b376f117217e4427f658190b0b558 for how we fixed this).

It is better to let the admin decide how to handle this, and use strict defaults.

I'm going to close this PR.

Reply:
Thanks for your contributions! @sathieu